core/src/templates/components/alert/alert.twig

```diff
@@ -19,13 +19,13 @@
  * - alert_body: Text details of the alert message.
  */
 #}
-<div {{ attributes }} class="su-alert {{ modifier_class }}" >
-  <div class="su-alert__body">
+<article {{ attributes }} class="su-alert {{ modifier_class }}" role="alert">
```

Comment:
Hi @cjwest , unfortunately I don't believe we can use `role='alert'` here. It implies that the element is a "live region" so more appropriate for dynamic alerts, which won't work well for a news type alert, e.g., "extreme heat on campus". As for the outside wrapper, I'm happy to change the `<section>` to a more generic `<div>`. Perhaps ask JB's opinion about whether to use article or div. She usually has good argument for that type of things :).

Reply:
Thank you @yvonnetangsu !!!

@JBCSU,
I'd love your $0.02 as I'm learning about how to use HTML5. 
What do you think, should we use `<article>` or `<div>` to wrap the alert component?

Reply:
This is new. I suddenly can't push my next commit. I get the message:
```
remote: Resolving deltas: 100% (7/7), completed with 5 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/v6-components-html5-alert.
remote: error: At least 1 approving review is required by reviewers with write access.
To github.com:SU-SWS/decanter.git
 ! [remote rejected] v6-components-html5-alert -> v6-components-html5-alert (protected branch hook declined)
error: failed to push some refs to 'git@github.com:SU-SWS/decanter.git'
```

Reply:
I do not think `<article>` is appropriate for an alert. An `<article>` should make sense in an RSS feed; I don't believe alerts would makes sense in that context.

`<aside>` is another option, but I think `<div>` is fine.